freqtrade/optimize/hyperopt_tools.py

```diff
@@ -267,6 +285,21 @@ def get_result_table(config: dict, results: list, total_epochs: int, highlight_b
         )
 
         stake_currency = config['stake_currency']
+
+        if has_drawdown:
+            trials['Max Drawdown'] = trials.apply(
+                lambda x: '{} {}'.format(
+                    round_coin_value(x['max_drawdown_abs'], stake_currency),
+                    '({:,.2f}%)'.format(x['Max Drawdown'] * perc_multi).rjust(10, ' ')
+                ).rjust(25 + len(stake_currency))
+                if x['Max Drawdown'] != 0.0 else '--'.rjust(25 + len(stake_currency)),
+                axis=1
```

Comment:
i do find the width of 25 a little wide - but i guess depending on the stake / drawdown amount, it'll be fine.